SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -12,6 +12,21 @@
 
 #include "SpatialConstants.generated.h"
 
+UENUM()
```

Comment:
Why is `ERPCType` not in `SpatialConstants` namespace anymore?

Reply:
It's a UENUM, so it has to be in global namespace.

Reply:
And it has to be a UENUM to be available from the editor UI.